WcaOnRails/db/migrate/20240112161627_create_translators_group.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+class CreateTranslatorsGroup < ActiveRecord::Migration[7.1]
+  # rubocop:disable Style/NumericLiterals
+  VERIFIED_TRANSLATORS_BY_LOCALE = {
+    "ca" => [94007, 15295],
+    "cs" => [8583],
+    "da" => [6777],
+    "de" => [870, 7121, 7139],
+    "eo" => [1517],
+    "es" => [7340, 1439],
+    "fi" => [39072],
+    "fr" => [277],
+    "hr" => [46],
+    "hu" => [368],
+    "id" => [1285],
+    "it" => [19667],
+    "ja" => [32229, 1118],
+    "kk" => [201680],
+    "ko" => [14],
+    "nl" => [1, 41519],
+    "pl" => [6008, 1686],
+    "pt" => [331],
+    "pt-BR" => [18],
+    "ro" => [11918],
+    "ru" => [140, 1492],
+    "sk" => [7922],
+    "sl" => [1381],
+    "sv" => [17503],
+    "th" => [21095],
+    "uk" => [296],
+    "vi" => [7158],
+    "zh-CN" => [9],
+    "zh-TW" => [38, 77608],
+  }.freeze
+  # rubocop:enable Style/NumericLiterals
+  def change
+    user_group = UserGroup.create!(name: 'Translators', group_type: :translators, is_active: true, is_hidden: true)
+    CreateTranslatorsGroup::VERIFIED_TRANSLATORS_BY_LOCALE.flat_map do |locale, user_ids|
+      user_ids.map do |user_id|
+        metadata = GroupsMetadataTranslators.create!(locale: locale)
+        UserRole.create!(
+          user_id: user_id,
+          group_id: user_group.id,
+          start_date: '1900-01-01',
```

Comment:
Hrm, I feel like there should be a more realistic start date. Can you do some (very basic! No need to dig super deep) `git blame` to find out when the feature was introduced? Or maybe look when the first `thewca-bot` PR was merged?

At the _very_ least, use the founding date of the WCA or something. It is highly improbable we had translators before the WCA even existed.

Reply:
First commit by `thewca-bot` is on 2017-01-30. I've given that for now.